albumentations/augmentations/transforms.py

```diff
@@ -841,9 +841,10 @@ class RandomShadow(ImageOnlyTransform):
     Args:
         shadow_roi: region of the image where shadows
             will appear. All values should be in range [0, 1].
-        num_shadows_lower: Lower limit for the possible number of shadows.
+        num_shadows_limit: Lower and upper limits for the possible number of shadows.
+        num_shadows_lower: Deprecated: Lower limit for the possible number of shadows.
             Should be in range [0, `num_shadows_upper`].
-        num_shadows_upper: Lower limit for the possible number of shadows.
+        num_shadows_upper: Deprecated: Lower limit for the possible number of shadows.
             Should be in range [`num_shadows_lower`, inf].
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_clarification):** Consider clarifying the deprecation message for better developer experience.

It might be helpful to include information on when `num_shadows_lower` and `num_shadows_upper` will be removed or if they will be removed in a future version.

```suggestion
        num_shadows_limit: Lower and upper limits for the possible number of shadows.
        num_shadows_lower: Deprecated: Lower limit for the possible number of shadows.
            Should be in range [0, `num_shadows_upper`]. This parameter will be removed in a future version.
        num_shadows_upper: Deprecated: Upper limit for the possible number of shadows.
            This parameter will be removed in a future version. Please use `num_shadows_limit` instead.
```